doc/api/process.markdown

```diff
@@ -70,6 +70,16 @@ Example of listening for `exit`:
     });
 
 
+## Event: 'message'
+
+* `message` {Object} a parsed JSON object or primitive value
+* `sendHandle` {Handle object} a Socket or Server object, or undefined
+
+Messages sent by
+[ChildProcess.send()](child_process.html#child_process_child_send_message_sendhandle)
```

Comment:
If links don't fit comfortably in 80 columns, you can put a `[ChildProcess.send()]: child_process.html#child_process_child_send_message_sendhandle` at the bottom and then you just write `[ChildProcess.send()]` here.